OWNERS

```diff
@@ -7,7 +7,7 @@ reviewers:
   - dcbw
   - JacobTanenbaum
   - juanluisvaladas
-  - pecameron
+  - trozet
```

Comment:
This breaks the sorting...

Reply:
also maybe add @fepan as an approver but not a reviewer (for emergency merges when no one else is around?)

Reply:
I didn't know it was sorted. Will do.